src/main/java/seedu/nutrigoals/logic/parser/ArgumentMultimap.java

```diff
@@ -25,10 +30,16 @@ public class ArgumentMultimap {
      * @param prefix   Prefix key with which the specified argument value is to be associated
      * @param argValue Argument value to be associated with the specified prefix key
      */
-    public void put(Prefix prefix, String argValue) {
+    public void put(Prefix prefix, String argValue) throws ParseException {
         List<String> argValues = getAllValues(prefix);
-        argValues.add(argValue);
-        argMultimap.put(prefix, argValues);
+        if (argValues.size() > 0) {
+            throw new ParseException(MESSAGE_MULTIPLE_TAGS_ERROR);
+        } else {
+            argValues.add(argValue);
+            argMultimap.put(prefix, argValues);
+        }
```

Comment:
Maybe the happy path could be made un-indented as suggested [here](https://nus-cs2103-ay2223s1.github.io/website/se-book-adapted/chapters/codeQuality.html#make-the-happy-path-prominent)?

For example:
```
if (argValues.size() > 0) {
    throw new ParseException(MESSAGE_MULTIPLE_TAGS_ERROR);
} 
argValues.add(argValue);
argMultimap.put(prefix, argValues);
```

Reply:
Noted, updated!